engine/apps/slack/scenarios/distribute_alerts.py

```diff
@@ -859,7 +921,9 @@ def process_signal(self, log_record: AlertGroupLogRecord) -> None:
         # Remove "memo" emoji from resolution note messages
         for message in alert_group.resolution_note_slack_messages.filter(added_to_resolution_note=True):
             try:
-                self._slack_client.reactions_remove(channel=message.slack_channel_id, name="memo", timestamp=message.ts)
+                self._slack_client.reactions_remove(
+                    channel=message.slack_channel_slack_id, name="memo", timestamp=message.ts
+                )
```

Comment:
this was a typo.

`ResolutionNoteSlackMessage` has a `slack_channel_slack_id` property which points to the Slack ID of the associated channel (ex. `C123FB4X`; `slack_channel_id` will actually point to the primary key ID of the associated `slack_channel` database record; ex. 14)